helm/alfresco-process-infrastructure/values.yaml

```diff
@@ -481,6 +481,9 @@ alfresco-deployment-service:
     image:
       # alfresco-deployment-service.applications.image.tag -- default tag for all images used in application
       tag: develop
+      pullPolicy: IfNotPresent
+      # alfresco-deployment-service.applications.image.pullPolicy -- pull policy for alfresco-deployment-service image
+      # @default -- do not pull new image if already exists on the running node
```

Comment:
we can remove this line probably, all people reviewing an helm chart pr know what pull policy is for

Reply:
ok sure